src/storage/src/engine.rs

```diff
@@ -379,6 +381,7 @@ impl<S: LogStore> EngineInner<S> {
             region.name(),
             region.id()
         );
+        increment_gauge!(crate::metrics::REGION_COUNT, 1.0);
```

Comment:
It seems that we only need to count in the `open_region` method, right?

Reply:
Or is it more reasonable to perform the counting in `RegionImpl#open`? This way, it will be consistent with `close` method.

Reply:
It seems we need to count both `open_region` and `create_region` because both create active region and add into the `RegionMap`.